compiler/rustc_session/src/options.rs

```diff
@@ -722,6 +710,15 @@ macro_rules! options {
                 None => false,
             }
         }
+
+        fn parse_split_debuginfo(slot: &mut Option<SplitDebuginfo>, v: Option<&str>) -> bool {
+            match v.and_then(|s| SplitDebuginfo::from_str(s).ok()) {
```

Comment:
This will ignore any parse errors.

Reply:
This does what other parsing functions do in this file.